react/PaymentSummary.tsx

```diff
@@ -19,49 +20,90 @@ const messages = defineMessages({
 const PaymentSummary: React.FC = () => {
   const {
     installmentOptions,
-    payment: { installments: installmentCount, paymentSystem },
+    paymentSystems,
+    payment: { installments: installmentCount, paymentSystem: paymentSystemId },
     cardLastDigits,
   } = useOrderPayment()
 
   const intl = useIntl()
 
+  const selectedPaymentPaymentSystem = paymentSystems.find(
```

Comment:
Is the repetition in selected**PaymentPayment**System intentional?

Reply:
yep, it would stand for the paymentSystem for the selected payment 😂 but I'm open to suggestions!

Reply:
I would go for `selectedPaymentSystem`. If it is the `paymentSystem` of the selected payment, then it is the selected payment system after all, right?

Reply:
:nazare: I think you're right, I'll change it